model-engine/model_engine_server/inference/vllm/vllm_server.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+import json
+from typing import AsyncGenerator
+
+import uvicorn
+from fastapi import BackgroundTasks, FastAPI, Request
+from fastapi.responses import Response, StreamingResponse
+from vllm.engine.arg_utils import AsyncEngineArgs
+from vllm.engine.async_llm_engine import AsyncLLMEngine
+from vllm.sampling_params import SamplingParams
+from vllm.utils import random_uuid
+
+TIMEOUT_KEEP_ALIVE = 5  # seconds.
```

Comment:
Do we need concurrency limiting too?

Reply:
concurrency limiting lives in http forwarder